main.go

```diff
@@ -194,6 +205,27 @@ func main() {
 		}
 	}()
 
+	if config.ChefAction {
+		// TODO: (@afiune) Re design a bit more to have different use-cases, maybe sub-commands?
+		//
+		// 1) Start the Chef-Load service for continous data:
+		//       chef-load start -config foo.conf
+		// 2) Load one time data with random fields:
+		//       (NODES):   chef-load generate nodes 100 -config foo.conf
+		//       (ACTIONS): chef-load generate actions 100 -config foo.conf
+		// 3) Initialize the chef-load conig
+		//       chef-load init
+		//
+		// For now we just have one flag -chef-action to land in this if
+		fmt.Printf("Loading 10 Chef Actions")
+		rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
I know this is the bit I should add, but I'm a little confused. Do I add this seed just once or on a scope where it is changed every time a node is generated?